packages/zoltan2/src/models/Zoltan2_GraphModel.hpp

```diff
@@ -205,12 +206,12 @@ class GraphModel : public Model<Adapter>
          will on return point to a StridedData object of weights.
        \return The number of ids in the edgeIds list.
    */
-  // Implied Vertex LNOs from getVertexList are used as indices to offsets
+  // Implied Vertex offset_t from getVertexList are used as indices to offsets
```

Comment:
The original comment is still correct; don't change it to offset_t.
Here we are talking about the indices into the offsets array; we are assuming the local numbering from the order in which the data is provided.